<launch>
	<arg name="images_path" default="/home/fnovotny/playground/flashcode_mbt/data/images/test-%010d.jpg" />
	<arg name="log_pattern" default="08d.jpg" />
	
	<node pkg="camera1394" type="camera1394_node" name="my_camera1394_node" args="_video_mode:=640x480_rgb8" >
	     <param name="camera_info_url" value="package://visp_auto_tracker/data/calibration.ini" type="string" />
	     <param name="frame_id" value="/map" type="string" />
	</node>


	<node pkg="ar_visp" type="ar_visp" name="recorder" args="camera_prefix:=/camera" output="screen">
	      <param name="tracker_log" type="string" value="$(arg log_pattern)"/>
	      <param name="pose_file" type="string" value="$(arg pose_file)"/>
	</node>

	<node name="ar_pose" pkg="ar_pose" type="ar_single" respawn="false">
	      <param name="marker_pattern" type="string" value="$(find ar_pose)/data/patt.qrwin"/>
	      <param name="kill_at_video_end" type="bool" value="true" />
	      <param name="marker_width" type="double" value="80.0"/>
	      <param name="marker_center_x" type="double" value="0.0"/>
	      <param name="marker_center_y" type="double" value="0.0"/>
	      <param name="threshold" type="int" value="100"/>
	      <param name="use_history" type="bool" value="true"/>

	      <remap from="/usb_cam/image_raw" to="/camera/image_raw" />
	      <remap from="/usb_cam/camera_info" to="/camera/camera_info" />

  	</node>

</launch>
